coral-schema/src/main/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverter.java

```diff
@@ -126,13 +126,20 @@ public Schema toAvroSchema(String dbName, String tableOrViewName) {
    * @return avro schema for a given sql query
    */
   // TODO: 2/3/22  to revisit whether we want to make it public
-  protected Schema toAvroSchema(String sql) {
+  public Schema toAvroSchema(String sql) {
```

Comment:
Is this method only used in testing? If so, I'm not sure of making it public.

Reply:
If the method is only used by the test in the same package, making it as package scope should be enough

Reply:
The test code that uses this is in a different package so it won't work.